pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
```

Comment:
It [directly mentions package.json](https://github.com/tldr-pages/tldr/blob/master/pages/common/npm.md), maybe we should do the same here as well.